services/backend/src/resolvers/RegularUser.studyLogs.ts

```diff
@@ -0,0 +1,14 @@
+import type { RegularUserResolvers } from '../generates/graphql'
+import { listStudyLogs } from '../modules/studyLog'
+
+export const studyLogsResolver: NonNullable<
+  RegularUserResolvers['studyLogs']
+> = async (regularUser, _, { user }) => {
+  if (!user) {
+    throw new Error('ログインされていません')
+  }
+  const studyLogs = await listStudyLogs(regularUser.id)
```

Comment:
ここクエリパターン使いたいね!そうしたらその後ソートさせなくて良くなるね

```suggestion
  const studyLogs = await listStudyLogs(regularUser.id, (collection) => collection.orderBy("studiedAt"))
```

Reply:
確かにそっちの方が簡潔ですね！ありがとうございます